packages/client/css/pages/account-settings.css

```diff
@@ -0,0 +1,97 @@
+.AccountSettings {
+  background: var(--gray-900);
+  max-width: 700px;
+
+  padding-right: 80px;
+  box-shadow: 0 0 16px -4px #00000044;
+}
+
+.AccountSettings-title {
+  margin-left: 180px;
+}
+
+.AccountSettings-input {
+  display: flex;
+  flex-direction: row;
+
+  & > label {
+    width: 150px;
+    line-height: 50px;
+
+    margin: 0;
+    margin-right: 14px;
+
+    text-align: right;
+  }
+
+  & > p {
+    line-height: 1.6;
+    margin-left: 13px;
+  }
+
+  & > .styled-button.no-bg {
+    color: var(--red);
+  }
+
+  &.\--avatar {
+    flex-wrap: wrap;
+
+    & > img {
+      border-radius: 4px;
+      width: 50px;
+      height: 50px;
+
+      margin-left: 24px;
+    }
+
+    & + p {
+      margin-left: 180px;
+      color: var(--gray-300);
+
+      & a {
+        color: var(--gray-100);
+      }
+    }
+  }
+
+.AccountSettings-submit {
+  display: block;
+
+  margin-top: 80px;
+  margin-left: auto;
+  margin-right: 0;
+  margin-bottom: 20px;
```

Comment:
Any reason not to consolidate these into a single `margin`?

Reply:
(Note the order would be `margin: 80px 0 20px auto`.)